jib-gradle-plugin/src/main/java/com/google/cloud/tools/jib/gradle/ContainerParameters.java

```diff
@@ -37,6 +37,7 @@
   private List<String> jvmFlags = Collections.emptyList();
   private Map<String, String> environment = Collections.emptyMap();
   @Nullable private List<String> entrypoint;
+  @Nullable private List<String> extraClasspath;
```

Comment:
Sorry, missed this one. Assign `= Collections.emptyList()`; and remove `@Nullable`, like the `environment` above.